configs/system/base.js

```diff
@@ -38,6 +38,19 @@ module.exports = {
         image: 'dashpay/sentinel:1.5.0',
       },
     },
+    tor: {
+      enable: true,
```

Comment:
Are you sure we want to have it enabled by default for all presets?

Reply:
No, I'm not sure. I am waiting for guidance from @PastaPastaPasta about when this service should be enabled

Reply:
Ok. I pretty sure we don't need it for the local network.